### PR TITLE
chore: upgrade Electron to latest v25 to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by mcmxcdev**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/HEAD/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change
- upgrade electron from v25.8.1 to 25.8.4

#### Motivation and Context
Adopts security vulnerability fixes patched in https://github.com/electron/electron/releases/tag/v25.8.2 and https://github.com/electron/electron/releases/tag/v25.8.4

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`pnpm prepare-code`)
- [x] `pnpm test` passes
- [x] I tested/previewed my changes locally

### CODE REVIEW

File: recipes [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit cdafac130fd454610d1fa2cb53b061b17b64e2ce
+Subproject commit 80f8496bf811a8fb93e9a961141c4868f66608fe
```
Comment (SpecialAro) [inline recipes]: Hi @mcmxcdev ! Thank you for this PR. I think you staged and committed a change in the recipes submodule (that is probably why the builds are failing)

**Review by SpecialAro — CHANGES_REQUESTED** [review]

File: recipes (hunk 1 above)
Comment (mcmxcdev) [reply recipes]: Ugh, had to rush, always happens to me unintentionally when I 'git add -A'.

I will be afk for a while, so could you remove it from the PR and get it merged?

Comment (mcmxcdev) [reply recipes]: Solved it by undoing the changes and force-pushing again in https://github.com/ferdium/ferdium-app/pull/1385/commits

**Review by Alphrag — APPROVED** [review]
Capture d’écran 2023-10-01 à 23 26 55

**Review by SpecialAro — APPROVED** [review]

**Comment by bukajsytlos** [comment]
any idea why using amd64 deb
image

**Comment by mcmxcdev** [comment]
@bukajsytlos Very strange, for me the same version release mentions a different electron version :thinking:

image

**Comment by bukajsytlos** [comment]
sry it's issue with arch packages using arch version of electron

**Pull request merged**